src/renderer/vt/VtSequences.cpp

```diff
@@ -434,6 +434,17 @@ using namespace Microsoft::Console::Render;
     return _Write("\x1b[?9001h");
 }
 
+// Method Description:
+// - Send a sequence to the connected terminal to switch to the alternate or main screen buffer.
+// Arguments:
+// - useAltBuffer: if true, switch to the malt buffer, otherwise to the main buffer.
+// Return Value:
+// - S_OK if we succeeded, else an appropriate HRESULT for failing to allocate or write.
+[[nodiscard]] HRESULT VtEngine::_SwitchScreenBuffer(const bool useAltBuffer) noexcept
+{
+    return _Write(useAltBuffer ? "\x1b[?1049h" : "\x1b[?1049l");

```

Comment:
This is technically only valid for the xterm-derived engines; wintelnet might B E E F

Reply:
True but I doubt that users of telnet.exe hit anything using the alt buffer...